src/pages/index.tsx

```diff
@@ -171,9 +172,7 @@ export const getStaticProps: GetStaticProps = async () => {
 
   return {
     props: {
-      posts: posts.data.postConnection.edges
-        ?.map((edge) => edge?.node)
-        .reverse(),
+      posts: convertTypesAndGetEdges(posts),
```

Comment:
Can you add similar logic to the statistics page, so it does not Tina pages as content?

Reply:
![image](https://github.com/1alexvash/AlexCodeBlog/assets/60061661/f9e7332a-9c8f-43dc-9fe7-f93510c652f6)
I have already done it in previous PR :) 
https://github.com/1alexvash/AlexCodeBlog/pull/167/files#diff-69659084985253a79b905760da9916f24601e74f38b847e577e7821e3b7aa047